src/core/packages/saved-objects/server/src/saved_objects_type.ts

```diff
@@ -201,7 +201,7 @@ export interface SavedObjectsType<Attributes = any> {
    *
    * When enabled, objects of this type can have an `accessControl` property containing:
    * - `owner`: The ID of the user who owns this object
-   * - `accessMode`: Optional access mode setting, currently supporting 'read_only'
+   * - `accessMode`: Access mode setting, currently supporting 'read_only' or 'default'.
```

Comment:
Rather under promise and over deliver! Experience has taught me using 'currently' anywhere leads to "can we add x, y, z" 
```suggestion
   * - `accessMode`: Access mode setting, supports 'read_only' or 'default'.
```